make/langtools/tools/propertiesparser/resources/templates.properties

```diff
@@ -50,19 +50,41 @@ factory.decl.method=\
     ' '*/\n\
     {0}public static {1} {2}({3}) '{'\n\
     {4}\n\
-    '}'
+    '}\n'
```

Comment:
This could probably just use text blocks and replacements - I'll leave that for another day.